src/game.cpp

```diff
@@ -702,6 +714,36 @@ class GameGlobalShaderConstantSetter : public IShaderConstantSetter
 			sunlight.b };
 		m_day_light.set(dnc, services);
 
+		// Lighting will also use location-based lighting in the future
+		// for smaller models so the sun color alone isn't sufficient
+		const float time_of_day = (float)m_client->getEnv().getTimeOfDayF();
```

Comment:
I asked about it in #minetest-dev, someone (possibly paramat?) said casts should not have spaces after them.